neptune/exceptions.py

```diff
@@ -59,9 +86,23 @@ def __init__(self):
         super(NoChannelValue, self).__init__('No channel value provided.')
 
 
-class LibraryNotInstalled(NeptuneException):
+class NeptuneLibraryNotInstalledException(NeptuneException):
     def __init__(self, library):
-        super(LibraryNotInstalled, self).__init__("Library {} is not installed".format(library))
+        message = """
+{h1}     
+----NeptuneLibraryNotInstalledException---------------------------------------------------------------------------------
+{end}
+Looks like library {library} wasn't installed.
+
+To install run:
+    {bash}pip install{library}{end}
```

Comment:
When I look at the usage in contrib, this won't work. It'll be `pip installKeras` (without a space even).

Reply:
Dropped this usage from contrib (and removed it there with `ModuleNotFoundError`. 
I don't think it made much sense to use custom NeptuneException for such usecase by the way.